tests/metadata-handler.js

```diff
@@ -32,16 +36,16 @@ describe('Metadata Handler', () => {
                     'username',
                 ];
                 _.each(types, (type) => {
-                    let msg = 'Fields of type ' + type + ' are not supported. Please define them manually.';
-                    expect(() => Meta.generateFieldValue({type: type})).to.throw(msg);
+                    let msg = `Fields of type ${type} are not supported. Please define them manually.`;
+                    expect(() => Meta.generateFieldValue({ type })).to.throw(msg);
                 });
             });
         });
 
         describe('unrecognized types', () => {
             it('should throw an error', () => {
                 let msg = 'Field type i_am_not_a_real_type is not recognized.';
-                expect(() => Meta.generateFieldValue({type: 'i_am_not_a_real_type'})).to.throw(msg);
+                expect(() => Meta.generateFieldValue({ type: 'i_am_not_a_real_type' })).to.throw(msg);
```

Comment:
Similar thing here like with `headers` and white space. White space like this doesn't look good to me when I am scanning the code.